content/docs/use-cases/data-registries.md

```diff
@@ -1,92 +1,87 @@
-[versioning of data and model files](/doc/use-cases/data-and-model-files-versioning),
-with commands such as `dvc add`. With the aim to enable reusability of these
-data and ML artifacts between different projects, DVC also provides commands
-like `dvc import` and `dvc get`. This means that your projects can depend on
-data from other DVC repositories, **similar to a package management systems for
-data science**.
+[versioning of data and model files](/doc/use-cases/data-and-model-files-versioning).
+DVC also enables cross-project [reusability](/doc/start/data-access) of these
+<abbr>data artifacts</abbr>. This means that your projects can depend on data
+from other DVC repositories — like a **package management system for data
+science**.
 
-![](/img/data-registry.png) _Data and models as code_
+![](/img/data-registry.png) _Data management middleware_
 
-This means we can build a <abbr>DVC project</abbr> dedicated to tracking and
-versioning _datasets_ (or any large files, directories, ML models, etc.). The
-repository would have all the metadata and history of changes in the different
-datasets. We could see who updated what and when, and use pull requests to
-update data, the same way we do with code.
+We can build a <abbr>DVC project</abbr> dedicated to versioning _datasets_ (or
+data features, ML models, etc.). The repository would have all the metadata and
+change history for the data it tracks. We could see who changed what and when,
+and use pull requests to update data like we do with code. This is what we call
+a **data registry** — data management _middleware_ between ML projects and cloud
+storage.
 
-This is what we call a **data registry** — a kind of data management
-_middleware_ between ML projects and cloud storage. Here are its advantages:
+Advantages of data registries:
 
-- Reusability: reproduce and organize _feature stores_ with a simple CLI
+- **Reusability**: reproduce and organize _feature stores_ with a simple CLI
   (`dvc get` and `dvc import` commands, similar to software package management
   systems like `pip`).
-- Persistence: [remote storage](/doc/command-reference/remote) (e.g. an S3
+- **Persistence**: [remote storage](/doc/command-reference/remote) (e.g. an S3
   bucket) controlled by the DVC registry improves data security. There are less
   chances someone can delete or rewrite an ML model, for example.
-- Storage optimization: centralize data
+- **Storage optimization**: centralize data
   [shared](/doc/use-cases/sharing-data-and-model-files) by multiple projects in
   a single location (distributed copies are possible too). This simplifies data
   management and optimizes space requirements.
-- Data as code: leverage Git workflows such as commits, branching, pull
+- **Data as code**: leverage Git workflows such as commits, branching, pull
   requests, reviews, and even CI/CD for your data and models lifecycle. Think
   "Git for cloud storage", but without ad-hoc conventions.
```

Comment:
Should it be "Codification" like in the upcoming Versioning use case?

Reply:
I'd like to keep `data as code` as well (to coin the term)

also Codification is a general term (applies to anything, not only data) - you will need to clarify the meaning in this case, means will have to rewrite it quite a bit (which is fine, but we don't have time for this)

unrelated: `but without ad-hoc conventions.` sounds a bit strange- I guess it's not easy to understand

Reply:
> keep data as code as well (to coin the term

So should we put it in the Versioning use case too?

> without ad-hoc conventions. sounds a bit strange

It does come out of nowhere. Should I remove it for now? I think we were referring to file names and that is explained in the Versioning one now.